ocean_lib/aquarius/aquarius.py

```diff
@@ -156,6 +156,28 @@ def validate_metadata(self, metadata: dict) -> Tuple[bool, Union[list, dict]]:
         parsed_response = response.json()
         return False, parsed_response
 
+    @enforce_types
+    def validate_asset(self, asset: V4Asset) -> Tuple[bool, Union[list, dict]]:
+        """
+        Validate the asset.
+
+        :param asset: conforming to the asset accepted by Ocean Protocol, V4Asset
+        :return: bool
+        """
+        asset_dict = asset.as_dictionary()
+
+        response = self.requests_session.post(
+            f"{self.base_url.replace('/v1/', '/')}/ddo/validate-remote",
+            data=json.dumps(asset_dict),
+            headers={"content-type": "application/json"},
+        )
+
+        if response.content == b"true\n":
```

Comment:
please use the decoded the binary response instead, which should be plain `True`. I think using response.json() is enough but you should be able to figure that out with pdb.

Reply:
i.e. parsed_response = response.json()
if parsed_response is True: return True, []
return False, parsed_response